cocos/3d/lights/directional-light-component.ts

```diff
@@ -446,6 +451,45 @@ export class DirectionalLight extends Light {
         }
     }
 
+    /**
+     * @en Enabled shadow advanced options
```

Comment:
What's `advanced options` mean? What advanced features?

Reply:
This option will improve quality and reduce performance. In the future we will simplify the shadow-related settings, collapsing some unnecessarily complex options into the advanced options

Reply:
Ok. What i mean is that, should add these information in docs.

Reply:
sure